src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import logo from '../assets/planet.png';
+import '../styles/header.css';
+
+const Header = () => (
+  <nav>
+    <div className="logoTitle">
+      <img src={logo} alt="space-traveler-logo" />
+      <h1>Space Travelers Hub</h1>
+    </div>
+    <div className="navLinks">
+      <Link to="/" className="links active-link">Rockects</Link>
+      <Link to="/missions" className="links">Missions</Link>
+      <Link to="/dragons" className="links">Dragons</Link>
```

Comment:
- [ ] As part of the project requirements, kindly make the active link underlined, right now the first link is underlined even if it's not the active one while the others are not underlined 👍 

![image](https://user-images.githubusercontent.com/50721479/205454539-ced0a311-03d4-4d95-aeb3-257f5d26ca5e.png)